docs/using/installing/index.rst

```diff
@@ -4,17 +4,27 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The following diagram illustrates the architecture of the cluster you are about to install:
 
-There is also a ``clusterhq-flocker-node`` package which is installed on each node in the cluster.
-It contains the services that need to run on each node.
+.. image:: images/installintro.png
+   :width: 75%
+   :alt: Diagram illustrating the Flocker CLI and control service and agents.
 
-.. note:: The ``clusterhq-flocker-node`` package is pre-installed by the :ref:`Vagrant configuration in the tutorial <tutvagrant>`.
+The CLI is installed on your local machine via the ``flocker-cli`` package which provides command line tools to control the cluster. 
+This package also includes the ``flocker-ca`` tool, which you must use to generate certificates for all the Flocker components.
```

Comment:
Personally I like to avoid "you" in documents. I guess that this is a style thing that you will decide upon. But "you must" is unpleasant to read. Perhaps in the context of "to do X you must first do Y" but my reaction to this is "MUST I?".